docs/source/developer_guide.md

```diff
@@ -296,20 +296,20 @@ Sometimes, we need to temporarily change the log pattern (eg: for reporting deci
 
 RAFT is a heavily templated library. Several core functions are expensive to compile and we want to prevent duplicate compilation of this functionality. To limit build time, RAFT provides a precompiled library (libraft.so) where expensive function templates are instantiated for the most commonly used template parameters. To prevent (1) accidental instantiation of these templates and (2) unnecessary dependency on the internals of these templates, we use a split header structure and define macros to control template instantiation. This section describes the macros and header structure.
 
-**Macros.** We define the macros `RAFT_COMPILED` and `RAFT_EXPLICIT_INSTANTIATE_ONLY`. The `RAFT_COMPILED` macro is defined by `CMake` when compiling code that (1) is part of `libraft.so` or (2) is linked with `libraft.so`. It indicates that a precompiled `libraft.so` is present at runtime.
+**Macros.** We define the macros `CUVS_COMPILED` and `CUVS_EXPLICIT_INSTANTIATE_ONLY`. The `CUVS_COMPILED` macro is defined by `CMake` when compiling code that (1) is part of `libraft.so` or (2) is linked with `libraft.so`. It indicates that a precompiled `libraft.so` is present at runtime.
```

Comment:
You're right, perhaps this whole section is obsolete by now. But also I've just noticed these changes are not relevant to any of the changed code in this PR, so I'll just remove the changeset, so we can fix this in a separate PR.

Reply:
I don't mean to make this your problem, btw. I think at some point we need to basicaly rewrite this developer guide so that it matches more closely to cuVS. RAFT is header only and we designed things a certain way for reusable primitives. cuVS is not header only and we've established some different programming paradigms as a result. I'll create an issue to revamp the developer guide eventually.